build/database/ProvisionDatabase_MVC.cs

```diff
@@ -0,0 +1,150 @@
+
+            AddUser("cmsadmin", "Administrator Administrator", password, new[] { "WebEditors", "WebAdmins" });
+            AddUser("abbie", "Abbie Andrea", password, new[] { "WebEditors", "WebAdmins" });
+            AddUser("eddie", "Eddie Elridge", password, new[] { "WebEditors" });
+            AddUser("erin", "Erin Ehrhardt", password, new[] { "WebEditors" });
+            AddUser("reid", "Reid Rezac", password, new[] { "WebEditors" });
+        }
+
+        private void AddUser(string userName, string fullName, string passWord, string[] roleNames)
+        {
+            if (UIUserProvider.GetUser(userName) == null)
+            {
+                var email = string.Format("epic-{0}@mailinator.com", userName);
+                IEnumerable<string> erros;
+                UIUserCreateStatus status;
+                var user = UIUserProvider.CreateUser(userName, passWord, email, null, null, true, out status, out erros);
+                UIRoleProvider.AddUserToRoles(user.Username, roleNames);
+
+                var profile = EPiServerProfile.Get(user.Username);
+                var nameParts = fullName.Split(' ');
+                profile["DisplayName"] = fullName;
+                profile["FirstName"] = nameParts[0];
+                profile["LastName"] = nameParts[1];
+                // E-mail must be part of profile properties to be resolved by QueryableNotificationUsersImpl
+                profile["Email"] = email;
+                profile.Save();
+            }
+        }
+
+        private void AddRole(string roleName, AccessLevel accessLevel, IContentSecurityRepository securityRepository)
+        {
+            if (!UIRoleProvider.RoleExists(roleName))
+            {
+                UIRoleProvider.CreateRole(roleName);
+
+                var permissions = (IContentSecurityDescriptor)securityRepository.Get(ContentReference.RootPage).CreateWritableClone();
+                permissions.AddEntry(new AccessControlEntry(roleName, accessLevel));
+
+                securityRepository.Save(ContentReference.RootPage, permissions, SecuritySaveType.Replace);
+                securityRepository.Save(ContentReference.WasteBasket, permissions, SecuritySaveType.Replace);
+            }
+        }
+
+
+        public UIUserProvider GetDeafultUserProvider()
+        {
+            UIUserProvider userProvider = null;
+            try
+            {
+                // Owin is not configured that becuase we have catch (membership provider  there is not problem)
+                ServiceLocator.Current.TryGetExistingInstance<UIUserProvider>(out userProvider);
+                return userProvider;
+            }
+            catch { }
+
+            // in the case of aspnet identity the rpovider is not in the service locator before owin is sets up then we create own.
+            var userManager = new ApplicationUserManager<ApplicationUser>(new UserStore<ApplicationUser>(new ApplicationDbContext<ApplicationUser>("EPiServerDB")));
```

Comment:
I would extract name of the connectionString from config element. `EPiServerDataStoreSection.Instance.DataSettings.ConnectionStringName`